engine-comparison/begin-experiment.sh

```diff
@@ -34,28 +34,12 @@ done
 gcloud compute scp fengine-configs/ ${INSTANCE_NAME}:~/input --recurse
 rm -r fengine-configs
 
-# These will frequently/usually be defined by the user
-JOBS=${JOBS:-8}
-N_ITERATIONS=${N_ITERATIONS:-5}
+CONFIG=${SCRIPT_DIR}/config
+echo "BMARKS=$1" > ${CONFIG}/bmarks.cfg
```

Comment:
So what is the intended process for running an experiment?  From what I understand:

1. gcloud init
2. Generate service account key
3. Modify parameters.cfg.
4. Specify benchmarks on command line

1 and 2 we don't have much control over, but it seems a little awkward from a user perspective to be specifying parameters both in the config file and on the command line.  Is there a compelling reason to do this?